sdk/core/azure-core/src/main/java/com/azure/core/implementation/jackson/JacksonVersion.java

```diff
@@ -87,11 +87,11 @@ private void checkVersion(SemanticVersion version, String packageName) {
             logger.error("Version '{}' of package '{}' is not supported (older than earliest supported version - `{}`), please upgrade.", version.getVersionString(), packageName, MIN_SUPPORTED_VERSION);
         }
 
-        if (version.getMajorVersion() > MAX_SUPPORTED_VERSION.getMajorVersion()) {
+        if (version.getMajorVersion() > MAX_SUPPORTED_MAJOR_VERSION) {
             logger.error("Major version '{}' of package '{}' is newer than latest supported version - '{}'.",
```

Comment:
With this change this error message doesn't make as much sense.

```suggestion
            logger.error("Major version '{}' of package '{}' is newer than latest supported major version - '{}'.",
```

Reply:
thanks!! It didn't make sense before that too :)